freeipa/src/main/resources/schema/app/20190516151400_add_tenant_to_stack.sql

```diff
@@ -0,0 +1,32 @@
+-- // Create Changelog
+
+-- Default DDL for changelog table that will keep
+-- a record of the migrations that have been run.
+
+-- You can modify this to suit your database before
+-- running your first migration.
+
+-- Be sure that ID and DESCRIPTION fields exist in
+-- BigInteger and String compatible fields respectively.
+
+ALTER TABLE stack
+    ADD tenant VARCHAR(255);
+
+CREATE UNIQUE INDEX tenant_uindex
```

Comment:
I think we need the 3rd unique index only.

Reply:
Rather than dropping this index, I think we can just remove the "UNIQUE" from it. This could aid the select performance.